packages/flutter_tools/bin/podhelper.rb

```diff
@@ -319,25 +322,44 @@ def flutter_install_plugin_pods(application_path = nil, relative_symlink_dir, pl
     # Keep pod path relative so it can be checked into Podfile.lock.
     relative = flutter_relative_path_from_podfile(symlink)
 
+    # If Swift Package Manager is enabled and the plugin has a Package.swift,
+    # skip from installing as a pod.
```

Comment:
Do you know off-hand what counts as "when plugins change" [here](https://github.com/flutter/flutter/blob/477ebd831c1253d2e19ac8084963f3088282ddd3/packages/flutter_tools/lib/src/macos/cocoapods.dart#L309-L314)? I'm wondering if we re-run `pod install` if the version of an existing plugin changes, or only when the set of plugins change. If the latter, it seems like we could have issues when an existing project's dependencies add SPM support.

(This may well be answered elsewhere in the PR, I'm just doing some initial skimming and asking questions as they come up.)

Reply:
We use `Fingerprinter` that checks various files. I added the generated Swift Package manifest as a path so that if it changes, `pod install` will also happen

https://github.com/flutter/flutter/blob/8c04fd74c8278ad400951098fc963e8bfd6282f0/packages/flutter_tools/lib/src/macos/cocoapod_utils.dart#L32-L49